projects/flow.pytorch.HyperparameterOptimize/source/labels.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import signac
+from flow import FlowProject
+
+
+class Project(FlowProject):
+    pass
+
+
+@Project.label
+def status_label(job):
+    check_list = ["train", "evaluation"]
+    check_results = ""
+    for check_point in check_list:
+        if job.doc.get(f"{check_point}_done") is not None:
+            check_results += f"{check_point}_completed, "
+        return check_results
```

Comment:
Faulty logic the last line should be deindented.
```suggestion
    return ", ".join([
        f"{check_point}_completed" for check_point in ("train", "evaluation")
        if job.doc.get(check_point + "_done", False)
    ])
```